### PR TITLE
[15.0][MIG]account_banking_mandate

### PR DESCRIPTION
**Pull request by manuelregidor**
Supersedes https://github.com/OCA/bank-payment/pull/883

### CODE REVIEW

**Comment by manuelregidor** [comment]
@ValentinVinagre @HaraldPanten

**Comment by HaraldPanten** [comment]
@MiquelRForgeFlow @pedrobaeza

**Review by HaraldPanten — APPROVED** [review]
Functional review 👍

**Review by ValentinVinagre — APPROVED** [review]
LGTM 👍🏻

File: account_banking_mandate/tests/test_invoice_mandate.py [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+            }
+        )
+        invoice._onchange_partner_id()
+
+        # Workaround: Odoo 15 does not allow the creation of new payment
+        # methods if not related to a module that inherits the function
+        # _get_payment_method_information from model account.payment.method
+        # Change pending of being applied when Odoo fixes this issue.
+
+        # pay_method_test = self.env["account.payment.method"].create(
```
Comment (ValentinVinagre) [inline account_banking_mandate/tests/test_invoice_mandate.py]: See here one example: https://github.com/odoo/odoo/blob/756b571d3bc72d47f9cf952d28e9765d1192dc39/addons/payment/tests/common.py#L21

**Review by ValentinVinagre — CHANGES_REQUESTED** [review]
Adapt the tests please.

File: account_banking_mandate/tests/test_invoice_mandate.py (hunk 1 above)
Comment (ValentinVinagre) [reply account_banking_mandate/tests/test_invoice_mandate.py]: Perfect, thanks 😄

**Review by ValentinVinagre — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**